app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsImageTagsFragment.kt

```diff
@@ -331,40 +328,39 @@ class SuggestedEditsImageTagsFragment : SuggestedEditsItemFragment(), CompoundBu
         csrfClient.request(false, object : CsrfTokenClient.Callback {
             override fun success(token: String) {
 
-                val claimObservables = ArrayList<ObservableSource<MwPostResponse>>()
-                val claimComments = ArrayList<String>()
+                val mId = "M" + page!!.pageId()
+                var claimStr = "{\"claims\":["
+                var commentStr = "/* add-depicts: "
                 for (label in acceptedLabels) {
-                    val claimTemplate = "{\"mainsnak\":" +
+                    if (claimStr.length > 16) {
+                        claimStr += ","
+                    }
+                    if (commentStr.length > 20) {
+                        commentStr += ","
+                    }
```

Comment:
Could you please add comments here to descrip the meaning of the numbers?

Reply:
You're right, that wasn't very clean. I've refactored it a bit to remove the numbers entirely.